h/buildext.py

```diff
@@ -35,13 +39,32 @@
 resolve = AssetResolver().resolve
 
 
-def build_extension_common(env, bundle_app=False):
+class Resolver(webassets.env.Resolver):
+    """A custom resolver for webassets which can resolve
```

Comment:
Thanks. Sorry about these issues. I've actually read the relevant PEPs now, I should have done that much sooner.